src/Boot/BootstrapManager.php

```diff
@@ -377,14 +380,20 @@ public function bootstrapValidate($phase)
     /**
      * Bootstrap to the specified phase.
      *
-     * @param string $bootstrapPhase
-     *   Name of phase to bootstrap to. Will be converted to appropriate index.
+     * @param \Consolidation\AnnotatedCommand\AnnotationData $annotationData
+     *   The annotation data from the command.
      *
      * @return bool
      *   TRUE if the specified bootstrap phase has completed.
+     *
+     * @throws \Exception
+     *   Thrown when an unknown bootstrap phase is passed in the annotation
+     *   data.
      */
-    public function bootstrapToPhase($bootstrapPhase)
+    public function bootstrapToPhase(AnnotationData $annotationData)
```

Comment:
The signature here feels less usable/convenient. One has to build an AnnotationData.